### DISCUSSION TITLE
Bitnet.cpp and Phi family models?

### DISCUSSION THREAD

**Discussion by JORGEGO** [microsoft/BitNet#86] [General]
Bitnet is very new to me, and what I'm saying probably doesn't make sense, but is it possible to use bitnet.cpp and apply it to a Microsoft phi model, so that it can run better on a windows CPU? If so, do you have a place where it explains what steps should be taken? Could the resulting model be used in semantic kernel?

**Comment by BradKML** [comment]
Related to other people wanting models to be quantized here https://github.com/microsoft/BitNet/discussions/234
